bfm/main/CMakeLists.txt

```diff
@@ -16,22 +16,20 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
-include(${CMAKE_INSTALL_PREFIX}/cmake/CMakeGlobal_Includes.txt)
-
 # ------------------------------------------------------------------------------
 # Targets
 # ------------------------------------------------------------------------------
 
 if(NOT WIN32)
     add_custom_command(OUTPUT bfvmcall_intel_x64.o
```

Comment:
this whole block was a hack because I had to compile BFM as a set of objects instead of as a static library. If this was fixed, we should compile this using the standard method instead of doing it by hand.